schemas/test_relationships_response.json

```diff
@@ -0,0 +1,71 @@
+{
```

Comment:
Should this be in the examples/ folder and/or should it replace https://github.com/ONSdigital/ons-schema-definitions/blob/v3/examples/eq_submission_example.json ?

Reply:
We refreshed the eq_submission_example schema and have now removed the test_relationships_schema.